contracts/english_auction_rev1.1.scilla

```diff
@@ -143,11 +159,17 @@ let make_error =
 
 contract EnglishAuctionOrderbook
 (
-  initial_contract_owner: ByStr20
+  initial_contract_owner: ByStr20,
+  collection_contract: ByStr20 with contract
```

Comment:
@sam-zilliqa can this be set to zero address for other applications which do not intend to use a brand collection? Just like allowlist_address can be zero